### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by ankushbhardwxj**
@narayanshivansh49 see this

### CODE REVIEW

File: Contract/Election.sol [hunk 1]
```diff
@@ -37,7 +37,7 @@ contract SampleVoting {
 
     function addCandidate (string memory _name) private {
         candidates[candidatesCount] = Candidate(candidatesCount, _name, 0);
-        candidatesCount ++;
```
Comment (shivansh-narayan) [inline Contract/Election.sol:40]: you changed the space ?

Comment (ankushbhardwxj) [reply Contract/Election.sol:40]: yeah.

Comment (shivansh-narayan) [reply Contract/Election.sol:40]: Ok will give you points....but in real projects or open-source don't request a pull for one space

**Review by shivansh-narayan — APPROVED** [review]

File: Contract/Election.sol (hunk 1 above)
Comment (ankushbhardwxj) [reply Contract/Election.sol:40]: Oh. okay !

**Pull request closed without merging**